src/pg/relations.cpp

```diff
@@ -62,4 +70,38 @@ void EstimateRelSize(Relation rel, int32_t *attr_widths, BlockNumber *pages, dou
 	PostgresFunctionGuard(estimate_rel_size, rel, attr_widths, pages, tuples, allvisfrac);
 }
 
+Oid
+GetRelidFromSchemaAndTable(const char *schema_name, const char *entry_name) {
+	List *name_list = NIL;
+	name_list = lappend(name_list, makeString(pstrdup(schema_name)));
+	name_list = lappend(name_list, makeString(pstrdup(entry_name)));
+	RangeVar *table_range_var = makeRangeVarFromNameList(name_list);
+	return RangeVarGetRelid(table_range_var, AccessShareLock, true);
+}
+
+bool
+IsValidOid(Oid oid) {
+	return oid != InvalidOid;
+}
+
+HeapTuple
+SearchSysCacheForRel(Oid oid) {
+	return SearchSysCache1(RELOID, ObjectIdGetDatum(oid));
+}
+
+bool
+IsValidHeapTuple(HeapTuple tuple) {
+	return HeapTupleIsValid(tuple);
+}
```

Comment:
You refactored the code so these functions aren't needed anymore, but you haven't removed them yet. (same for `ReleaseSysCache`.

Reply:
Ugh good catch.